xbmc/music/Song.h

```diff
@@ -40,6 +40,13 @@ class CGenre
 
 class CFileItem;
 
+ // Struct to store chapter information from mp3 files (could be used for m4b or mka files too)
+struct chapterDetails {
+  std::string chapterName;
+  unsigned int startTime;
+  unsigned int endTime;
```

Comment:
you should default init the values here, so you can be sure never to have them unintialized.

```
  unsigned int startTime{0};
  unsigned int endTime{0};
  ```

Reply:
Done